hack/generators/release-controllers/generate-release-controllers.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, git_clone_dir):
         self.rc_deployment_domain = 'apps.ci.l2s4.p1.openshiftapps.com'
         self.rc_release_domain = 'svc.ci.openshift.org'
         self.rc_deployment_namespace = 'ci'
-        self.arches = ('x86_64', 's390x', 'ppc64le')
+        self.arches = ('x86_64', 's390x', 'ppc64le', 'arm64')
```

Comment:
Are you sure about this?  Given the use of `x86_64` here, If `uname -m` (or equivalent) is being used, then it should be `aarch64`.

Reply:
The `x86_64` is/was used initially and has since been deprecated by `amd64`...
https://github.com/openshift/release/blob/fa305c3cb9f3047797ccb4cad0aa9ab5bba75b1e/hack/generators/release-controllers/generate-release-controllers.py#L121-L125